main/java/by/stub/yaml/YamlParser.java

```diff
@@ -197,6 +198,10 @@ private boolean isPairKeyEqualsToYamlNodeFile(final String pairKey) {
    private Object loadFileContentFromFileUrl(final Object rawPairValue) throws IOException {
       final String filePath = StringUtils.objectToString(rawPairValue);
       try {
+         if(FileUtils.doesFilePathContainTemplateTokens(new File(filePath))) {
```

Comment:
@radek1st Is this `if` block really necessary here? What I mean is:
We are constructing a new File object in this function anyways. So, the `FileUtils.uriToFile` can be changed not to throw `IOException` in case if you are constructing new File using tokenized path.

I hope I am clear